modules/cache/cache_test.go

```diff
@@ -54,26 +55,25 @@ func TestGetString(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "", data)
 
-	// data, err = GetString("key", func() (string, error) {
-	// 	return "some data", nil
-	// })
-	// assert.NoError(t, err)
-	// assert.Equal(t, "", data)
-	// Remove("key")
+	data, err = GetString("key", func() (string, error) {
+		return "some data", nil
+	})
+	assert.NoError(t, err)
+	assert.Equal(t, "", data)
+	Remove("key")
 
 	data, err = GetString("key", func() (string, error) {
 		return "some data", nil
 	})
 	assert.NoError(t, err)
 	assert.Equal(t, "some data", data)
 
-	// data, err = GetString("key", func() (string, error) {
-	// 	return "", fmt.Errorf("some error")
-	// })
-	// assert.NoError(t, err)
-	// assert.Equal(t, "some data", data)
-
-	// TODO: uncommented code works in IDE but not with go test
+	data, err = GetString("key", func() (string, error) {
+		return "", fmt.Errorf("some error")
+	})
+	assert.NoError(t, err)
+	assert.Equal(t, "some data", data)
+	Remove("key")
```

Comment:
This test could fail if somehow this thread idles for 30 nanoseconds (`setting.CacheService.TTL = 30`)?

Reply:
Isn't that in milliseconds or even seconds?
But yes, theoretically it is possible that this test fails randomly.

Reply:
TTL is [`time.Duration`](https://pkg.go.dev/time#Duration)
> A Duration represents the elapsed time between two instants as an int64 nanosecond count.

Reply:
These tests should cover the "is in the cache and does not need to be computed" case, so the TTL should be irrelevant I think. A simple `setting.CacheService.TTL = 24 * time.Hour` should prevent random failures.